app/helpers/application_helper.rb

```diff
@@ -302,8 +302,8 @@ def edit_inline(object, property, options = {})
     editable(object, property, {:type => type, :title => title, :value => value, :class => klass, :source => select_values, :url => update_url, :placeholder => placeholder}.compact)
   end
 
-  def documentation_url(section = "", options = {})
-    main_app.external_link_url(options.merge(type: 'manual', params: { section: section }))
+  def documentation_url(section = nil, type: 'manual', **options)
+    main_app.external_link_url(type: type, section: section, params: options)
```

Comment:
`external_link_url` should receive only two parameters: `type` and `options`.  I think splitting the `section` out of the options will cause a different output

Reply:
Are you sure? The route is includes the section:
https://github.com/theforeman/foreman/blob/df3a0b0d970b229887e2d371568d16c8143c5aae/config/routes.rb#L558
Now I'll immediately admit my knowledge of Rails routing may be lacking, but I don't see a difference between `type` and `section` there, other than that `section` is optional.

Reply:
Well, you are correct:
```
[9] pry(main)> external_link_path(type: 'foo', section: 'bar', zzz: 'qqq')
=> "/links/foo/bar?zzz=qqq"
```
As long as it works - I'm good :)

Reply:
I didn't check if passing it without a section also works, but I'm assuming it does.

Reply:
It does. When no section is passed, the final url does not contain a fragment. 
So, as expected.